torchdynamo/resnet18.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2021 Google LLC
```

Comment:
Let's call this directory "torchdynamo_poc". Once we have more clarity on the longer-term path we can fold this into the main python package.